src/features/Cart/hooks/useCartCheckStatus.ts

```diff
@@ -0,0 +1,7 @@
+import { useCartContext } from '../context/CartProvider';
+
+export const useCartCheckStatus = () => {
+  const { cartItems } = useCartContext();
+  const allChecked = cartItems.every((item) => item.isChecked);
+  return { allChecked };
+};
```

Comment:
이 경우에는 유틸함수로 표현해도 좋았을 것 같아요!

```suggestion
export const isAllChecked = (cartItems) => {
  return cartItems.every((item) => item.isChecked);
};
```

요로코롬..